src/FSAL/FSAL_CORTXFS/conf/nfs_setup.sh

```diff
@@ -99,6 +100,35 @@ function motr_lib_init {
 	[ $? -ne 0 ] && die "Failed to Initialise motr_lib fs_meta index"
 }
 
+function prepare_pnfs_conf {
+IFS=',' read -ra ADDR <<< "$pNFS_DATA_SERVER"
+count=0
+rm -rf /etc/ganesha/pNFS.conf
+echo "[pNFS]" >> /etc/ganesha/pNFS.conf
+echo "  pNFS_Role = $pNFS_ROLE" >> /etc/ganesha/pNFS.conf
+
+if [ "$pNFS_ROLE" != "PNFS_DISABLED" ]; then
+pNFS_ENABLED=true;
+echo "pNFS_ENABLED is $pNFS_ENABLED"
+fi
+
+if [ "$pNFS_ROLE" == "MDS" ] || [ "$pNFS_ROLE" == "BOTH" ]; then
+for i in "${ADDR[@]}"; do
+let "count= count+1"
+echo "  DS$count = $i:2049" >> /etc/ganesha/pNFS.conf
+done
+
+if [ "$pNFS_ROLE" == "BOTH" ]; then
+let "count= count+1"
+echo "  DS$count = $(hostname -i):2049" >> /etc/ganesha/pNFS.conf
```

Comment:
We should support configurable port number for the NFS servers and a task can be created if not there already.

Reply:
@sachinpunadikar , is this a possibility.If so, will create a ticket for this under ToDo

Reply:
I don't think, configurable port is need, actually that will complicate the things.
So 2049 would be the fixed port number.

Reply:
Actually most of the application servers on TCP provides this option to user. These are due to various reasons, such as firewall, interop with older clients, security etc. But I think it is fine as of now, in future if needed it can be checked.